go/vm/lfvm/interpreter.go

```diff
@@ -87,6 +86,14 @@ func (c *context) SignalError(err error) {
 	c.status = ERROR
 }
 
+func (c *context) isBerlin() bool {
```

Comment:
Just a though: how about calling this `isBerlinOrLater()`?

Reply:
In some way, the previous version that used conditions like `c.revision >= vm.R09_Berlin` captured the right thought and it is quite readable. Maybe the functions should be removed again? Wdyt?